docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,303 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a27a38f1-393d-49f9-92c3-a8634bfb1a60",
+   "metadata": {},
+   "source": [
+    "The `LayerError.error` attribute of the noise learning result contains the generators and error rates of the fitted Pauli Lindblad model, which has the form:\n",
+    "\n",
+    "$$ \\Lambda(\\rho) = \\exp{\\sum_j r_j \\left(P_j \\rho P_j^\\dagger - \\rho\\right)} $$\n",
+    "\n",
+    "where the $r_j$ are the `LayerError.rates` and $P_j$ are the Pauli operators specified in `LayerError.generators`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ebb4b7d-ef45-4996-bf5a-125d0831ebca",
+   "metadata": {},
+   "source": [
+    "## Noise learning options\n",
+    "\n",
+    "There are a few options available to input into a `NoiseLearner` object, can be set with when instantiated. These are encapsulated by the `qiskit_ibm_runtime.options.NoiseLearnerOptions` class and include the ability to specify the maximum layers to learn, number of randomizations, and the twirling strategy, among others. Refer to the API documentation about [`NoiseLearnerOptions`](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.NoiseLearnerOptions) for more detailed information.\n",
+    "\n",
+    "Below is a simple example showing how to use the `NoiseLearnerOptions` in a `NoiseLeaner` experiment:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8f35d68a-b459-4f1c-ae44-a01bbe700ed0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Build a GHZ circuit\n",
+    "circuit = QuantumCircuit(10)\n",
+    "circuit.h(0)\n",
+    "circuit.cx(range(0, 9), range(1, 10))\n",
+    "# Choose a backend to run on\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy()\n",
+    "\n",
+    "# Transpile the circuit for execution\n",
+    "pm = generate_preset_pass_manager(backend=backend, optimization_level=3)\n",
+    "circuit_to_run = pm.run(circuit_to_learn)\n",
+    "\n",
+    "# Instantiate a noise learner options object\n",
+    "learner_options = NoiseLearnerOptions(\n",
+    "    max_layers_to_learn = 3,\n",
+    "    num_randomizations = 32,\n",
+    "    twirling_strategy = \"all\"\n",
+    ")\n",
+    "\n",
+    "# Instantiate a NoiseLearner object and execute the noise learning program\n",
+    "learner = NoiseLearner(mode=backend, options=learner_options)\n",
+    "job = learner.run([circuit_to_run])\n",
+    "noise_model = job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "694b0e38-7aeb-48e0-8598-b64cff395bc2",
+   "metadata": {},
+   "source": [
+    "## Input noise model to primitive\n",
+    "\n",
+    "The noise model learned on the circuit can also be used as an input to the `EstimatorV2` primitive implemented in Qiskit IBM Runtime. This can be passed into the primitive a few different ways. For example, one can pass the noise model to the `estimator.options` attribute directly"
```

Comment:
```suggestion
    "The noise model learned on the circuit can also be used as an input to the `EstimatorV2` primitive implemented in Qiskit IBM Runtime. This can be passed into the primitive a few different ways. For example, you can pass the noise model to the `estimator.options` attribute directly"
```

I think we normally use the term "you" instead of "one"